packages/serverpod/lib/src/server/log_manager/log_manager.dart

```diff
@@ -254,6 +254,17 @@ class SessionLogManager {
 
   final Lock _openLogLock = Lock();
 
+  /// Opens the log for a session immediately if logging for all sessions is
+  /// enabled. Otherwise, it will be opened when something is logged or the
+  /// session is closed. Called automatically when a session is created.
+  @internal
+  Future<void> openLog(Session session) async {
+    var settings = _settingsForSession(session);
```

Comment:
Question: is this checked in some other place as well? Should those two be centralized in some way? Any risk for behaviour deviation?